apps/web/components/booking/BookingListItem.tsx

```diff
@@ -86,6 +87,13 @@ function BookingListItem(booking: BookingItemProps) {
     },
   });
 
+  const { data: query } = trpc.viewer.teams.getMembershipbyUser.useQuery({
```

Comment:
running this query for each booking item dosen't feel very performant to me, can we include this data in bookings/get.handler somehow?